test/unit/stats_test.lua

```diff
@@ -0,0 +1,24 @@
+local expirationd = require("expirationd")
+local t = require("luatest")
+local g = t.group("stats")
+
+local helpers = require("test.helper")
+
+g.before_all(function()
+    helpers.init_spaces(g)
+end)
+
+g.after_each(function()
+    helpers.truncate_spaces(g)
+end)
+
+function g.test_stats_simple()
+    expirationd.start("clean_all", g.tree.id, helpers.is_expired_true)
+    local stats = expirationd.stats("clean_all")
+    t.assert_equals(stats, {
```

Comment:
Don't you want to test each available counter? Currently this test only checks that `expirationd.stats` method just exists.

Reply:
Sure, will do.